statler_api/views.py

```diff
@@ -13,13 +15,18 @@ def healthCheck(request):
     return HttpResponse("The API is alive.")
 
 
-def getPlayList(request):
+def getPlayList(request, play_list_id):
     """called when a GET request is sent to /api/play-list/
     returns json of the list of plays
     see https://docs.djangoproject.com/en/1.8/topics/serialization/"""
-    #TODO: get an actual PlayList object instead of all plays
 
-    aPlayDictList = [vars(Play(x)) for x in PlayDAO.objects.all()]
+    if play_list_id == "all":
+        # /api/play_lists/all/ will automatically return all plays
+        aPlayList = PlayDAO.objects.all();
+    else:
+        aPlayList = get_object_or_404(PlayListDAO, url_title=play_list_id).plays.all()
+
+    aPlayDictList = [vars(Play(x)) for x in aPlayList]
```

Comment:
We can't just grab all the plays, because that blows away the order within the list. We need to serialize the nodes, including (potentially null) rank. Converting the base PlayListDAO to a PlayList should result in the object we need to serialize.